osu.Server.Queues.ScoreStatisticsProcessor/Commands/Maintenance/RecalculateTotalUserPerformanceCommand.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Dapper;
+using McMaster.Extensions.CommandLineUtils;
+using osu.Server.QueueProcessor;
+using osu.Server.Queues.ScoreStatisticsProcessor.Models;
+using osu.Server.Queues.ScoreStatisticsProcessor.Processors;
+
+// ReSharper disable InconsistentNaming
+namespace osu.Server.Queues.ScoreStatisticsProcessor.Commands.Maintenance
+{
+    [Command("recalculate-total-user-performance", Description = "Process all users' total performance")]
+    public class RecalculateTotalUserPerformanceCommand
+    {
+        public async Task<int> OnExecuteAsync(CancellationToken cancellationToken)
+        {
+            var processor = new UserTotalPerformanceProcessor();
+
+            using (var db = DatabaseAccess.GetConnection())
+            {
+                Stopwatch sw = new Stopwatch();
+
+                foreach (var score in await db.QueryAsync<SoloScore>("SELECT id FROM scores WHERE legacy_score_id IS NULL"))
+                {
+                    var score2 = db.QuerySingle<SoloScore>($"SELECT id, user_id, ruleset_id FROM scores WHERE id = {score.id}");
+                    if (cancellationToken.IsCancellationRequested)
+                        break;
+
+                    sw.Restart();
+
+                    processor.ApplyToUserStats(score2.ToScoreInfo(), new UserStatsOsu { user_id = (int)score2.user_id }, db, null!);
+
+                    Console.WriteLine($"Recalculating {score2.user_id} took {sw.ElapsedMilliseconds:N0} ms");
+                }
```

Comment:
Is this supposed to be here? I don't even know what this is attempting to do because it's making very little sense. Outer loop is over every non-imported score, inner query... fetches two more columns of the same row, and then something is happening with `ApplyToUserStats()` but the ruleset is... hardcoded by specifying `UserStatsOsu`? Very confused. I don't think this ever has an actual use for anything.

Reply:
This was just used to benchmark the performance of the processor. I think it can be removed because as you say, it's likely useless.